testsettings.py

```diff
@@ -98,6 +98,23 @@ def _set_logging_levels(levels):
     'loggers': {},
 }
 
+# Define an aaa-data database if its not already defined
+# This is necessary because REPORTING_DATABASES references aaa-data.
```

Comment:
Why do other values in `REPORTING_DATABASES` that are not in `DATABASES` (for example, `icds-ucr`) not need this special treatment? Why are they not causing the same failure that `aaa-data` did in my local test setup?

Reply:
I think `REPORTING_DATABASES` are used to map sqlalchemy engine_id -> django db alias. in this dictionary all the icds-ucr databases are aliased to the django db allias default which exists